i18n-util/src/LocaleUtil.js

```diff
@@ -0,0 +1,43 @@
+const LOCALE_COOKIE_KEY = 'okta_user_lang';
+
+const getLocale = () => {
+  const supportedLocales = [
+    'en',
```

Comment:
These map to the languages supported within h.o.c. `ja` is still in progress and might change to `ja-JP` in the future.

Reply:
We already differentiate en-us and en-uk for www.okta.com.    Is en really international english or is it US english?

Reply:
It is en-US. I believe that will also change from `en` to `en-US` in the future. These are mapped to [folder](https://github.com/okta/okta-help/tree/gh-pages/en) names currently where the content lives

Reply:
Since the locale determined by the browser will include the country code, we may resolve to the invalid locale using this logic.

Instead, could we map the language codes to a directory path?

Reply:
After doing some more testing found that some browsers send the locale value as `en`(Chrome) and some send it as `en-US`(Firefox). Updated logic to support both `en` and `en-US` and added a new map that converts a locale to a folder name that is supported. Once the folder names are updated to have language code in the future, we can remove that code.